lib/index.ts

```diff
@@ -1,9 +1,10 @@
-import * as colors from 'colors/safe';
 import * as fs from 'fs';
 import * as path from 'path';
 import * as semver from 'semver';
 import ajv, { ValidateFunction } from 'ajv';
 import addFormats from 'ajv-formats';
+import colors from 'colors/safe';
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
 // @ts-ignore no typings for json-source-map
```

Comment:
Maybe replace this with our own type definition for this library?

Something like:
```
declare module 'json-source-map' {
    export function parse(json: string): Object;
}
```

Could for example be stored in `types/json-source-map` with a reference in the compilerOptions:
`"typeRoots": ["./node_modules/@types", "./types"]`

Reply:
It has been added, please take a look.